src/main/java/org/entando/kubernetes/service/digitalexchange/BundleUtilities.java

```diff
@@ -0,0 +1,20 @@
+package org.entando.kubernetes.service.digitalexchange;
+
+import io.fabric8.zjsonpatch.internal.guava.Strings;
+import org.entando.kubernetes.model.debundle.EntandoDeBundle;
+
+public class BundleUtilities {
+
+    public static String getBundleVersionOrFail(EntandoDeBundle bundle, String version) {
+       
+        String versionToFind = version;
+        if ( ! (versionToFind.matches("\\d+\\.\\d+") || versionToFind.matches("\\d+\\.\\d+\\.\\d+"))) {
```

Comment:
This needs a comment. What is happening here?

Reply:
Basically this is to test support both versions formatted as versions - e.g. 2.0, 3.1.1, etc - and "meta" versions which are already included in npm - i.e. "latest" version